.github/workflows/simulator-tests.yml

```diff
@@ -0,0 +1,41 @@
+# Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
+# See the file LICENSE for licensing terms.
+
+name: Simulator Tests
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  simulator-tests:
+    runs-on:
+      labels: ubuntu-20.04-32
```

Comment:
If we are always running this, I think we should use `ubuntu-latest` or we should put this behind `run all ci`: https://github.com/ava-labs/hypersdk/blob/db1a98e691fd47c4db39650adbc72ff694470517/.github/workflows/tokenvm-release.yml#L18